src/renderer/protocol-handler/app-handlers.ts

```diff
@@ -0,0 +1,46 @@
+import { addClusterURL } from "../components/+add-cluster";
+import { clusterSettingsURL } from "../components/+cluster-settings";
+import { extensionsURL } from "../components/+extensions";
+import { landingURL } from "../components/+landing-page";
+import { preferencesURL } from "../components/+preferences";
+import { clusterViewURL } from "../components/cluster-manager/cluster-view.route";
+import { LensProtocolRouterRenderer } from "./router";
+import { navigate } from "../navigation/helpers";
+import { clusterStore } from "../../common/cluster-store";
+import { workspaceStore } from "../../common/workspace-store";
+
+export function bindProtocolAddRouteHandlers() {
+  LensProtocolRouterRenderer
+    .getInstance<LensProtocolRouterRenderer>()
+    .addInternalHandler("/preferences", ({ search: { highlight }}) => {
+      navigate(preferencesURL({ fragment: highlight }));
+    })
+    .addInternalHandler("/landing", () => {
+      navigate(landingURL());
+    })
```

Comment:
There's not really a landing page any more, just the overview of the current workspace...we could add a path to a specific workspace overview, by id, something like:
```suggestion
    .addInternalHandler("/landing", () => {
      navigate(landingURL());
    })
    .addInternalHandler("/landing/:workspaceId", ({ pathname: { workspaceId } }) => {
      workspaceStore.setActive(workspaceId);
      navigate(landingURL());
    })
```